packages/thirdparty-widgets/src/Map/index.tsx

```diff
@@ -50,8 +50,315 @@ Brizy.registerComponent({
                           value: "Chisinau",
                         },
                       },
+                      {
+                        id: "zoom",
+                        label: "Zoom",
+                        type: "slider",
+                        default: {
+                          value: 15,
+                        },
+                        config: {
+                          min: 1,
+                          max: 21,
+                        },
+                      },
+                    ],
+                  },
+                  {
+                    id: "filters",
+                    label: "Filters",
+                    options: [
+                      {
+                        id: "filter",
+                        type: "filters",
+                        default: {
+                          saturation: 100,
+                          brightness: 100,
+                          contrast: 100,
+                        },
+                        states: [NORMAL, HOVER],
+                        selector: "{{WRAPPER}}:hover .brz-ui-ed-iframe",
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+          {
+            id: "toolbarColor",
+            type: "popover",
+            config: {
+              size: "medium",
+              title: "Colors",
+              icon: {
+                style: {
+                  backgroundColor: "#000",
+                },
+              },
+            },
+            devices: "desktop",
+            position: 2,
+            options: [
+              {
+                id: "tabsColor",
+                type: "tabs",
+                tabs: [
+                  {
+                    id: "tabBackground",
+                    label: "Background",
+                    options: [
+                      {
+                        id: "background",
+                        type: "backgroundColor",
+                        selector: "{{WRAPPER}}:hover .brz-ui-ed-iframe",
+                        states: [NORMAL, HOVER],
+                      },
```

Comment:
need to return default value again for this

Reply:
I discussed with Sergiu and decided that is ok how is now